packages/cli/src/cli.ts

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+'use strict';
+
+import { runCli } from './lib/core/yargs/index.js';
+
+await runCli();
```

Comment:
I am not sure its logical to have a file that does this and only this.

Would it not be better to move the yargs parsing directly here?